run.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-cd `dirname $0`
+cd "$(dirname "$0")"
```

Comment:
`$()` command substitution is not a bashism at all, it's mandated by POSIX, see http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_03